### DISCUSSION TITLE
Automate Api in Findomain Configure file

### DISCUSSION THREAD

**Discussion by salooali** [Findomain/Findomain#153] [Q&A] [answered]
How can I configure findomain config file with API's so that I don't have to rewrite a full command like 
findomain_securitytrails_token="YourAccessToken" findomain -(options)
for each API. I want to use maximum API's and I don't want to write an individual command. I want all of the APIs working at the same time and give me combined result. I hope you understand my question 
I'm using Kali Linux version 2021.1
 Thank you.

**Comment by Edu4rdSHL** [comment, marked as answer]
Hello, please see the [configuration-examples](https://github.com/Findomain/Findomain/tree/master/config_examples), you need to simply create the configuration file, put the API keys there and then use `findomain -t target.com -c /path/to/config_file` or, if you want you can just name the config file `findomain.toml` (for example) and run Findomain directly in the same directory, it will read the configuration file even if `-c /path/to/config_file` is not specified.

**Reply by salooali** [reply]
Yes sir, It works Thank you for saving my life. I'm trying to use more API's but for now, I just get SecurityTrails,Facebook, and Virustotal with apikey API's could you kindly suggest more that are publicly available. I've checked the API's that are used by findomain but didn't get the API key.

**Reply by Edu4rdSHL** [reply]
Check https://github.com/Findomain/Findomain#how-it-works, then click on the API that you want to use for documentation. Only APIs marked with `*` or `**` can use API tokens.

**Reply by salooali** [reply]
Yes, I've checked these APIs, Thank you for your guidance.
